apps/etterlatte-saksbehandling-ui/client/src/components/behandling/attestering/oppsummering/Avbrutt.tsx

```diff
@@ -0,0 +1,105 @@
+import { isSuccess, mapApiResult } from '~shared/api/apiUtils'
+import Spinner from '~shared/Spinner'
+import { ApiErrorAlert } from '~ErrorBoundary'
+import { Oppgavestatus } from '~shared/types/oppgave'
+import { formaterDato } from '~utils/formatering/dato'
+import { KopierbarVerdi } from '~shared/statusbar/KopierbarVerdi'
+
+export const Avbrutt = ({ behandlingsInfo }: { behandlingsInfo: IBehandlingInfo }) => {
+  const [oppgaveForBehandling, hentOppgaveForBehandling] = useApiCall(hentOppgaverMedReferanse)
+  const [saksbehandlerNavn, hentNavnForIdent] = useApiCall(hentNavnforIdent)
+
+  useEffect(() => {
+    hentOppgaveForBehandling(behandlingsInfo.behandlingId)
+  }, [])
+
+  useEffect(() => {
+    if (isSuccess(oppgaveForBehandling)) {
+      const avbrutteOppgaver = oppgaveForBehandling.data.filter((oppgave) => oppgave.status === Oppgavestatus.AVBRUTT)
+      if (avbrutteOppgaver.length) {
+        const riktigOppgave = avbrutteOppgaver[0]
+        if (!riktigOppgave.saksbehandler?.navn) {
+          if (riktigOppgave.saksbehandler?.ident) {
+            hentNavnForIdent(riktigOppgave.saksbehandler?.ident)
+          }
+        }
+      }
+    }
+  }, [oppgaveForBehandling.status])
+
+  return (
+    <SidebarPanel $border>
+      <VStack gap="2">
+        <div>
+          <Heading size="small">{formaterBehandlingstype(behandlingsInfo.type)}</Heading>
+          <Heading size="xsmall">Avbrutt</Heading>
+        </div>
+
+        <Box paddingInline="2 0">
+          <HStack gap="2">
+            <SakTypeTag sakType={behandlingsInfo.sakType} size="small" />
+            <UtenlandstilknytningTypeTag utenlandstilknytningType={behandlingsInfo.nasjonalEllerUtland} size="small" />
+          </HStack>
+        </Box>
+        <VStack gap="2" justify="space-between">
+          {mapApiResult(
+            oppgaveForBehandling,
+            <Spinner label="Henter oppgave med saksbehandler" />,
+            () => (
+              <ApiErrorAlert>Kunne ikke hente oppgave for behandling</ApiErrorAlert>
+            ),
+            (oppgaver) => {
+              const oppgaveravbrutt = oppgaver.filter((oppgave) => oppgave.status === Oppgavestatus.AVBRUTT)
+              if (oppgaveravbrutt.length) {
+                const foersteavbrutteoppgave = oppgaveravbrutt[0]
+                const hentetNavn = isSuccess(saksbehandlerNavn) ? saksbehandlerNavn.data : undefined
+                return (
+                  <div>
+                    <Label size="small">Saksbehandler</Label>
+                    <Detail>
+                      {hentetNavn ||
+                        foersteavbrutteoppgave.saksbehandler?.navn ||
+                        foersteavbrutteoppgave.saksbehandler?.ident}
+                    </Detail>
+                  </div>
+                )
+              } else {
+                return <Detail>Ingen oppgave med status avbrutt funnet</Detail>
+              }
+            }
+          )}
```

Comment:
Flytt logikken for å finne første avbrutte oppgave ut av render loopen.

Reply:
Også er det ting i lowercase som burde være camelcase

Reply:
oooog bytt ut styled-compoent `<Detail />` med aksel komponent

Reply:
Hvilken?

Reply:
Hvilke lowercase ting?

Reply:
fixd